### PR TITLE
Burn

### PR DESCRIPTION
**Pull request by alexplevako**

### CODE REVIEW

File: .github/workflows/docker-hardfork-ci.yml [hunk 1]
```diff
@@ -45,12 +38,13 @@ jobs:
         -t "scorum/${NET_NAME}:0.${HF_VERSION}.0.${GIT_COMMIT}" .
 
     - name: Push docker image to registry
+      if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master'
```
Comment (margarita-markova) [inline .github/workflows/docker-hardfork-ci.yml:41]: why do you use master branch in build-testnet job?

File: .github/workflows/docker-hardfork-ci.yml [hunk 2]
```diff
@@ -3,19 +3,12 @@ name: Docker Hardfork CI
...
-      - 'libraries/chain/hardfork.d/*'
-      - '.github/workflows/docker-hardfork-ci.yml'
-
+      - '*'
 
 jobs:
   build-testnet:
     runs-on: self-hosted
 
-    if: github.ref == 'refs/heads/develop'
```
Comment (margarita-markova) [inline .github/workflows/docker-hardfork-ci.yml:17]: why not?

Comment (alexplevako) [reply .github/workflows/docker-hardfork-ci.yml:17]: I would like to run CI on push to feature branch.

File: .github/workflows/docker-hardfork-ci.yml (hunk 1 above)
Comment (alexplevako) [reply .github/workflows/docker-hardfork-ci.yml]: okay, lets build testnet image only from develop branch.

**Review by margarita-markova — APPROVED** [review]

**Pull request merged**